assets/src/edit-story/components/checklist/checklistContent/priorityChecks.js

```diff
@@ -74,7 +74,6 @@ export function PriorityChecks({
         <StoryTitleLength />
         <StoryMissingExcerpt />
         <StoryPosterAttached />
-        {/* TODO: #8129 this overlaps alot with aspect ratio, do we need both? */}
         <StoryPosterPortraitSize />
         <StoryPosterAspectRatio />
```

Comment:
My understanding from https://github.com/google/web-stories-wp/issues/8129#issuecomment-875980334 was to just merge these two components.

Could call it `StoryPosterSize` and check for dimensions & aspect ratio in there.

Reply:
sounds good, on friday it had seemed a good idea to keep them separate so we could pull the aspect ratio one out eventually since it's impossible to mess that up now but i agree with you. updating :)